HealthGPT/HealthGPTAppDelegate.swift

```diff
@@ -30,24 +30,19 @@ class HealthGPTAppDelegate: SpeziAppDelegate {
             }
             HealthDataInterpreter()
             HealthDataFetcher()
-            SecureStorage()
+            KeychainStorage()
         }
     }
 
 
     private var healthKit: HealthKit {
         HealthKit {
-            CollectSamples(
-                [
-                    HKQuantityType(.stepCount),
-                    HKQuantityType(.activeEnergyBurned),
-                    HKQuantityType(.appleExerciseTime),
-                    HKQuantityType(.bodyMass),
-                    HKQuantityType(.heartRate),
-                    HKCategoryType(.sleepAnalysis)
-                ],
-                deliverySetting: .manual()
-            )
+            CollectSample(.stepCount, start: .manual)
```

Comment:
We should probably use [`RequestReadAccess`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/1.1.3/documentation/spezihealthkit/requestreadaccess) here 👍

Reply:
The README could be improved to explain the difference better. I made a PR here: https://github.com/StanfordSpezi/SpeziHealthKit/pull/55